.coveragerc

```diff
@@ -3,3 +3,12 @@ source = cla_backend
 omit =
         *migrations*
         cla_backend/settings/*
+        ./venv/*
+        *tests*
+        *apps.py
+        *manage.py
+        *__init__.py
```

Comment:
This one gives me pause as I suspect we do have real code in some `__init__.py` files that I'd ideally want to see covered.

Is there a particular downside to having these covered, even when they're only the shebang etc?

Reply:
There isnt, made sure to check and removed from the omit list.